wrdcld/main.py

```diff
@@ -86,7 +86,7 @@ def fill_next_word(word, available_rectangles, image, font, frequency):
         options.append("vertical")
 
     if not options:
-        print(f"skipping word '{word}', couldn't find a good rectangle")
+        # print(f"skipping word '{word}', couldn't find a good rectangle")
```

Comment:
ℹ️ Idea (not for this PR perhaps): We move this to a logger, and add a `silent: bool` flag to the main function that allows people to toggle this output on or off.